doc/sphinx/practical-tools/coqide.rst

```diff
@@ -321,34 +342,43 @@ The top of the *Shortcuts* section lets you select the allowed modifiers
 that can be selected for the listed menus.  (The changes won't appear until
 you close and reopen the Preferences dialog.)
 
-*Misc*
+*Misc* – to be documented
 
 .. _user-configuration-directory:
 
 Preferences and key bindings are saved in the user configuration directory,
 which is ``$XDG_CONFIG_HOME/coq`` if the environment variable ``$XDG_CONFIG_HOME``
 is set.  If the variable isn't set, the directory is ``~/.config/coq`` on Linux
 and `C:\\Users\\<USERNAME>\\AppData\\Local\\coq` on Windows.
-Preferences are in the file "coqiderc" and key bindings are in the file "coqide.keys".
+Preferences are in the file `coqiderc` and key bindings are in the file `coqide.keys`.
+
+.. _coqide_key_bindings:
 
 .. _key_bindings:
 
 Key bindings
 ~~~~~~~~~~~~
 
+As explained just above, the *Edit/Preferences/Shortcuts* panel
+offers buttons to modify in a few clicks the key bindings for a whole menu.
+Here is a screenshot of the panel:
+
+.. image:: ../_static/coqide-preferences-shortcuts.png
+   :alt: CoqIDE preferences dialog, Shortcuts section
+
 Each menu item in the GUI shows its key binding, if one has been defined,
 on the right-hand side.  Typing the key binding is equivalent to selecting
-the associated item from the menu.
-A GTK+ accelerator keymap is saved under the name ``coqide.keys`` in
-the :ref:`user configuration directory<user-configuration-directory>`.
-You can modify the key binding ("accelerator") for a menu entry by
-going to the corresponding menu item without releasing the
-mouse button, pressing the keys you want for the new binding and then releasing
-the mouse button.
+the associated item from the menu.  On some systems, you can modify the
+key binding ("accelerator") for a menu entry by going to the corresponding
+menu item without releasing the mouse button, pressing the keys you want
+for the new binding and then releasing the mouse button.
 
-Alternatively, you can edit the file directly.  Make sure there are no
-CoqIDE processes running while you edit the file.  (CoqIDE creates or
-overwrites the file when it terminates, which may reorder the lines).
+Alternatively, you can edit the configuration file directly.
+A GTK+ accelerator keymap is saved under the name `coqide.keys` in
```

Comment:
```suggestion
Key bindings are saved in the file `coqide.keys` in
```

This file was just mentioned 25 lines above...

Reply:
OK, fixed.